.env.production

```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_API_URL=https://forma-devnet-modularspring.onrender.com/
```

Comment:
don't add production value. leave it empty for now as bridge will launch before modularium